cmd/orgs.go

```diff
@@ -215,6 +233,148 @@ func orgsRemove(ctx *cli.Context) error {
 	return nil
 }
 
+func orgsMembersListCmd(ctx *cli.Context) error {
+
+	// Use "member" team name to bypass specific teams
+	teamName := "member"
+
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	client := api.NewClient(cfg)
+	c := context.Background()
+
+	var getMembers sync.WaitGroup
+	getMembers.Add(2)
+
+	var org *envelope.Org
+	var orgs []envelope.Org
+	var team envelope.Team
+	var teams []envelope.Team
+	var memberships []envelope.Membership
+	var oErr, tErr, mErr, sErr error
+
+	// Retrieve the org name supplied via the --org flag.
+	// This flag is optional. If none was supplied, then
+	// orgFlagArgument will be set to "". In this case,
+	// prompt the user to select an org.
+	orgFlagArgument := ctx.String("org")
+
+	if orgFlagArgument == "" {
+		// Retrieve list of available orgs
+		orgs, oErr = client.Orgs.List(c)
+		if oErr != nil {
+			return oErr
+		}
+
+		idx, _, oErr := SelectOrgPrompt(orgs)
+		if oErr != nil {
+			return oErr
+		}
+
+		if idx == promptui.SelectedAdd {
+			// COME BACK TO THIS
+			// Error condition, unsure how to handle
+		}
+
+		org = &orgs[idx]
+	}
+
+	go func() {
```

Comment:
One thought I had (and totally  up for discussion) is whether the cognitive overhead of doing the lookup for the session concurrently with the lookup for org, team, and members is worth it. The session is actually cached inside the daemon, so it should be very fast anyhow!

Reply:
It would drastically simplify this code as we wouldn't need the waitgroup or handle the multi error scenario.

Reply:
I agree with avoiding concurrency here, I think the gain probably doesn't outweigh the confusion and the error handling is ugly. Also, in the event that no flag is used, there is a blocking API call anyway, followed up by the Go routines afterward, so I'm awkwardly mixing paradigms here. This was based on a similar function in teams.go that used the concurrent approach but to simplify things I'm happy to remove it.